src/include/ndpi_protocol_ids.h

```diff
@@ -155,7 +155,7 @@ typedef enum {
   NDPI_PROTOCOL_YOUTUBE               = 124,
   NDPI_PROTOCOL_SKYPE_TEAMS           = 125,
   NDPI_PROTOCOL_GOOGLE                = 126,
-  NDPI_PROTOCOL_RPC                   = 127,
+  NDPI_PROTOCOL_MS_RPCH               = 127,
```

Comment:
Could you add an entry in `doc/protocols.rst`, please (in order)? Basically, with the same info that you already reported in the commit message.
Other than that, it makes a lot of sense. Thanks

Reply:
BTW, any chance that you have a pcap example with this kind of traffic? It seems quite difficult to find...

Reply:
> BTW, any chance that you have a pcap example with this kind of traffic? It seems quite difficult to find...

Unfortunately, I don't have a pcap example, but I heard that this protocol is used by MS Exchange.

Reply:
Ok, I found the protocol specs  and it looks like this check can be a little bit extended. See[ 2.1.1.1.1.1 RPC Connect Request](https://winprotocoldoc.blob.core.windows.net/productionwindowsarchives/MS-RPCH/%5bMS-RPCH%5d.pdf)